### ISSUE TITLE
Group annotations

### ISSUE THREAD

**Issue by ghost** [ashare80/TSClusterMapView#23]
I have a couple of annotations which should be clustered always together and not with other annotations. For instance I have an application which shows annotations for certain regions of a city. It would be great if all the annotations in "Manhattan" would join together and the annotations of "Brooklyn" go into a new cluster. Therefor something like a "group ID" could be set while creating the annotations. Is something like this possible or would it change too much the way the cluster algorithm works?

**Comment by AdamShare** [comment]
@architector1 Yeah this was something that was on my roadmap.  It's definitely doable just haven't had the time to make the switch. Right now all the cluster-able annotations live in one tree and basically like you say you would just have multiple cluster trees and manage them by an ID.

**Comment by AdamShare** [comment]
@architector1 When you get a chance try out the grouped cluster annotation feature on branch feature/multi-tree. Should work pretty well if the annotations are grouped in different regions of the map like you were looking to do.
